src/endpoints/sources.js

```diff
@@ -391,26 +391,3 @@ function receiveSubresource(subresourceExtractor) {
  * @param {express.request} req
  * @param {express.response} res
  */
-
-
-export {
-  setSource,
-
-  setDataset,
-  canonicalizeDataset,
-  ifDatasetExists,
-  getDataset,
-  putDataset,
-  deleteDataset,
-  optionsDataset,
-
-  setNarrative,
-  ifNarrativeExists,
-  getNarrative,
-  putNarrative,
-  deleteNarrative,
-  optionsNarrative,
-
-  sendDatasetSubresource,
-  sendNarrativeSubresource,
-};
```

Comment:
Ahh, I think it's a bit of a misstep to switch from a clear export list like this to scattered per-symbol exports. To me, some files make more sense with per-symbol exports and some make more sense with a single export block at the bottom.

My suggestion in https://github.com/nextstrain/nextstrain.org/pull/735#discussion_r1401216668 was not intended to mean "use per-symbol exports everywhere", but I see now that you proposed that in response and I thumbs upped it without realizing.

Reply:
Thanks for following up – I misunderstood your initial comment and took it further than your intended scope of "import + export". New PR which should align better: #834